types/office-js-preview/index.d.ts

```diff
@@ -136073,6 +136328,22 @@ declare namespace PowerPoint {
          * [Api set: PowerPointApi 1.8]
          */
         readonly columnIndex: number;
+        /**
+         * Specifies the horizontal alignment of the table cell.
```

Comment:
I think this refers to the alignment of the _text_ within the cell. The "alignment of the cell" is kind of puzzling. 
```suggestion
         * Specifies the horizontal alignment of the text in the table cell.
```